### DISCUSSION TITLE
2009/05/18/labview%E8%AE%BA%E5%9D%9B%E3%80%81%E5%8D%9A%E5%AE%A2%E5%92%8C%E7%BE%A4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#262] [Announcements]
# 2009/05/18/labview%E8%AE%BA%E5%9D%9B%E3%80%81%E5%8D%9A%E5%AE%A2%E5%92%8C%E7%BE%A4

如果在学习过程中遇到问题，可以考虑通过互联网来寻求帮助。把相关的问题张贴在相关的论坛或高手博客上，便可得到别的的帮助。

https://blog.qizhen.xyz/2009/05/18/labview%E8%AE%BA%E5%9D%9B%E3%80%81%E5%8D%9A%E5%AE%A2%E5%92%8C%E7%BE%A4

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yechung
2009年12月5日 11:48 编辑
我們是一家醫療器材設計與生產的公司，現正徵求LabView高手負責專案工作。可以是專職也可顧問方式合作，如有興趣請跟我聯絡。 YC Wang Yc.Wang@shl-tw.com

回复
